src/views/Stake/components/StakeArea/components/RebaseTimer/hooks/useNextRebaseDate.ts

```diff
@@ -15,27 +14,6 @@ export const useNextRebaseDate = () => {
   };
 };
 
-/**
- * @returns JS Date for next warmup completion
- */
-export const useNextWarmupDate = () => {
-  const { data: secondsToRebase, isSuccess: rebaseSuccess } = useNextRebase();
-  const { data: warmupEpochs, isSuccess: warmupSuccess } = useWarmupPeriod();
-  const { data: epoch, isSuccess: epochSuccess } = useEpoch();
-  const isSuccess = rebaseSuccess && warmupSuccess && epochSuccess;
-
-  const parsedSeconds = parseBigNumber(secondsToRebase || BigNumber.from("0"), 0);
-  const epochLengthSeconds = parseBigNumber(epoch?.length.toBigNumber() || BigNumber.from("0"), 0);
-  const warmupLength = parseBigNumber(warmupEpochs?.toBigNumber() || BigNumber.from("0"), 0);
```

Comment:
prev code was always add 2 full epochs to your warmup, rather than using your expiry